modules/local/annotation/environment.yml

```diff
@@ -0,0 +1,9 @@
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - python=3.10
+  - pip=24.3.1
+  - pip:
+      - sopa>=2.1.0
+      - tangram-sc>=1.0.4
```

Comment:
please pin the exact version. same for the other environment.ymls https://nf-co.re/docs/guidelines/components/modules#use-of-conda-directive